sapphire/corsika/blocks.py

```diff
@@ -6,14 +6,14 @@
 
 Author: Javier Gonzalez <jgonzalez@ik.fzk.de>
 """
-
+from __future__ import division
 import struct
 import math
 
 import numpy
 
-import units
-import particles
+from sapphire.corsika import units
+from sapphire.corsika import particles
```

Comment:
Ouch! This is a mistake, I didn't properly understand python 3 imports when I patched this. `import .units` is fine :)